cmd/test/cmd.go

```diff
@@ -279,11 +280,12 @@ func doRun(ctx context.Context, options *Options, ioCtrl *io.Controller, k8sLogg
 				// resources in the environment, so including it to set the env vars in the remote-run
 				External: manifest.External,
 			},
-			Manifest: manifest,
-			Command:  remote.TestCommand,
+			Manifest:                    manifest,
+			Command:                     remote.TestCommand,
+			ContextAbsolutePathOverride: path.Clean(fmt.Sprintf("%s/%s", cwd, test.Context)),
 		}
 
-		ioCtrl.Logger().Infof("Executing test for: %s", name)
+		oktetoLog.Information("Executing '%s'", name)
```

Comment:
I guess you are using the global variable because you couldn't find how to show it with the other one:
```suggestion
		ioCtrl.Out().Information("Executing '%s'", name)
```

Reply:
```
type *"github.com/okteto/okteto/pkg/log/io".OutputController has no field or method Information
```